8Knot/assets/color.css

```diff
@@ -289,31 +289,6 @@ div.dark-card {
   background-color: var(--color-border);
 }
 
-/* =============================================================================
-   DROPDOWN COMPONENTS
-============================================================================= */
-
-/* Dark Dropdown Box Styling */
-.dark-dropdown .Select-control,
-.dark-dropdown .Select-control:hover {
-  background-color: var(--multiselect-hover-bg)!important;
-  border-color: var(--multiselect-hover-bg)!important;
-  color: white !important;
-  min-width: 150px !important;
-}
-
-/* Dark Dropdown Placeholder Styling */
-.dark-dropdown .Select-placeholder,
-.dark-dropdown .Select-value-label {
-  color: white !important;
-}
-
-/* Fix for the vertical bar issue */
```

Comment:
this should not be removed, this impacts the date range picker

Reply:
ah right. fixed!